x-pack/plugins/lists/server/services/exception_lists/duplicate_exception_list.ts

```diff
@@ -96,10 +85,16 @@ export const duplicateExceptionListAndItems = async ({
     });
     itemsToBeDuplicated = [...itemsToBeDuplicated, ...transformedItems];
   };
+  const savedObjectPrefix = getSavedObjectType({ namespaceType });
+  const filter = includeExpiredExceptions
+    ? []
+    : [
+        `(${savedObjectPrefix}.attributes.expire_time > "${new Date().toISOString()}" OR NOT ${savedObjectPrefix}.attributes.expire_time: *)`,
```

Comment:
This isn't necessarily a comment for your PR but we should perhaps centralize this filter string since it's used just about every place we have to filter for/against expired exceptions and probably will be going forward. I know we [have this](https://github.com/elastic/kibana/blob/ef218cd70423916763e5622d04029e722d209be2/packages/kbn-securitysolution-list-utils/src/helpers/index.ts#L924) for an array of `SavedObjectType`'s but nothing else.

Reply:
Good point! I can probably do a follow up with this. My only hesitation doing it here is trying not to increase the number of files touched since it's already much more than anticipated.